tests/test_user_secrets.py

```diff
@@ -49,7 +49,7 @@ def set_request(self):
                 _request['headers'] = self.headers
 
             def get_response(self):
-                return {"Secret": secret}
+                return {'result': {'secret': secret, 'secretType': 'refreshToken', 'secretProvider': 'google'}, 'wasSuccessful': True}
```

Comment:
Could you add a test with 'wasSuccessfull' : 'false' as well?

Reply:
Done.